Sources/sourcekit-lsp/SourceKitLSP.swift

```diff
@@ -137,6 +150,18 @@ struct SourceKitLSP: ParsableCommand {
   )
   var indexPrefixMappings = [PathPrefixMapping]()
 
+  @Option(
+    name: .customLong("compilation-db-search-path", withSingleDash: false),
+    parsing: .unconditionalSingleValue,
+    help:
+      "Specify a relative path where SourceKit should search for `compile_commands.json` or `compile_flags.txt` relative to the root of a workspace. Multiple search paths may be specified by repeating this option."
+  )
+  var compilationDatabaseSearchPaths = try! [
+    // These defaults match the behavior of clangd
+    RelativePath(validating: "."),
+    RelativePath(validating: "build"),
```

Comment:
It may make sense to pull these out and make the search paths "additional" so the defaults persist even in the presence of custom search paths.

Reply:
Yes, I think that would make sense. IMO the search precedence should be user-defined search paths, `.`, `build`.

Reply:
Done.